backends/common.go

```diff
@@ -0,0 +1,9 @@
+package backends
+
+type PBCKeyNotFoundError struct {
```

Comment:
Any reason for calling this `PBCKeyNotFoundError` rather than simply `KeyNotFoundError`?

Reply:
none in particular. Will modify so it's easier to read